registry/templates/create_bucket

```diff
@@ -1,3 +1,4 @@
+{{ if exists "/deis/store/gateway" }}
```

Comment:
Doing this means you can't have a bucket created for `stateless` clusters that use `s3` for baking - This particular script is a little confused and needs love to make it work for both scenarios. #4729 cleans up a lot of that, does it achieve what you are looking for?